tests/e2e/examples/source/openjdk-sb-postgresql/pom.xml

```diff
@@ -0,0 +1,188 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.example</groupId>
+    <artifactId>demo</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>Spring Boot - CRUD Demo</name>
+    <description>Spring Boot - CRUD Demo</description>
+
+    <properties>
+        <awaitility.version>3.1.0</awaitility.version>
+        <postgresql.version>9.4.1212</postgresql.version>
+        <rest-assured.version>3.1.0</rest-assured.version>
+        <snowdrop-bom.version>1.5.16.SP1</snowdrop-bom.version>
```

Comment:
Why not use `1.5.17`? ;)

Reply:
Good catch! I just generated a project from our generator which I guess hasn't been updated

Reply:
Fixed